### PR TITLE
GET | paintings artist

### PR DESCRIPTION
**Pull request by Mu7ammadAbed**
- I handled `/paintings/:artistId` route which send all paintings data.

- I made a function for `/paintings/:artistId` route.

- I made a database query which SELECT all rows from painting table WHERE their `artist_id` matches the entered id.

### CODE REVIEW

File: server/controllers/getArtistPaints.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  try {
+    const { artistId } = req.params;
+    if (artistId > 0) {
+      const { rows } = await getArtistPaintsQuery(artistId);
+      if (rows.length !== 0) {
+        res.json({ statusCode: 200, data: rows });
+      } else {
+        res.status(404).json({
+          statusCode: 404,
+          message: "Sorry There's no artist for this id",
```
Comment (LinaYahya) [inline server/controllers/getArtistPaints.js:13]: There's no painting for this artist :smiley:

File: test/paintings.test.js [hunk 2]
```diff
@@ -21,3 +21,44 @@ describe('Get all Paintings', () => {
       });
   });
 });
+
+describe('Get artist paints by id', () => {
+  test('Route /paintings/1 status 200, json header, data.name = ca-wiki ', (done) => {
```
Comment (LinaYahya) [inline test/paintings.test.js:26]: data.name = ca-wiki !! :wink:

File: server/controllers/getArtistPaints.js (hunk 1 above)
Comment (Mu7ammadAbed) [reply server/controllers/getArtistPaints.js:13]: You are right, thank you *.*

File: test/paintings.test.js (hunk 2 above)
Comment (Mu7ammadAbed) [reply test/paintings.test.js:26]: Oops, I didn't pay attention for this haha Lol, Thank you anyway *.*

Comment (Mu7ammadAbed) [reply test/paintings.test.js:26]: Please check if every thing is correct now, and what you mentioned before has been solved.

**Pull request merged**